Client/Frontend/Strings.swift

```diff
@@ -118,8 +118,16 @@ extension Strings {
     public static let RecentlyClosedTabsButtonTitle = NSLocalizedString("HistoryPanel.RecentlyClosedTabsButton.Title", value: "Recently Closed", comment: "Title for the Recently Closed button in the History Panel")
     public static let RecentlyClosedTabsPanelTitle = NSLocalizedString("RecentlyClosedTabsPanel.Title", value: "Recently Closed", comment: "Title for the Recently Closed Tabs Panel")
     public static let FirefoxHomePage = NSLocalizedString("Firefox.HomePage.Title", value: "Firefox Home Page", comment: "Title for firefox about:home page in tab history list")
+    
+    // History MO
+    public static let Today = NSLocalizedString("Today", comment: "History tableview section header")
```

Comment:
nit: Comment should be explicit, i.e. "History tableview section header for today" as this will help translators and same changes for yesterday, last week and last month